webapp/portlet/src/main/webapp/vue-apps/general-settings/components/branding/form/BorderRadiusSelector.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="d-flex mt-8">
+    <v-btn 
+      text
+      small
+      @click="setBorderRadius(0)">
+      {{ $t('generalSettings.widgetAndAppStyle.square') }}
+    </v-btn>
+    <v-slider
+      v-model="defaultBorderRadius"
+      :thumb-size="24"
+      tick-size="2"
+      thumb-label="always"
+      ticks
+      min="0"
+      max="20"
+      step="4"
+      color="primary"
+      @change="$emit('input', defaultBorderRadius)" />
+    <v-btn
+      text
+      small
+      @click="setBorderRadius(20)"> 
+      {{ $t('generalSettings.widgetAndAppStyle.rounded') }} 
+    </v-btn>
+  </div>
+</template>
+<script>
+export default {
+  props: {
+    borderRadius: {
+      type: String,
+      default: null,
+    },
```

Comment:
Use `v-model` with `this.$emit('input', this.borderRadius);` when the selected `borderRadius` changes. (use `watch` as well to reset value of `this.borderRadius = this.value`)

```suggestion
    value: {
      type: String,
      default: null,
    },
```

Reply:
done